ugvc/mrd/ppmSeq_utils.py

```diff
@@ -1397,7 +1414,7 @@ def plot_strand_ratio_category_concordnace(
         ("All reads", "Only reads where end was reached"),
         (df_category_concordance, df_category_concordance_no_end_unreached),
     ):
-        df_plot = df_plot.unstack().droplevel(0, axis=1)
+        df_plot = df_plot.to_frame().unstack().droplevel(0, axis=1)
```

Comment:
Again- if you need to_frmae() then it is not a df. Consider renaming to be more clear

Reply:
it is after I convert and unstack it